docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -768,6 +768,76 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_getSbundleStats
+
+The `flashbots_getSbundleStats` JSON-RPC method returns stats for a single sbundle. You must provide bundleHash, and the signing address must be the same as the one who submitted the bundle.
+Only bundles not containing backruns are supported
```

Comment:
Might not be immediately clear what we mean by this. Could clarify a bit: sbundle stats are only visible for bundles that do contain other users' private transactions. Ie. you cannot view sbundle stats on a bundle which backruns another user in MEV-Share.

(Feel free to reword, I just think it might need a bit more explanation).